src/components/interaction-designer/flow-editors/FlowEditor.vue

```diff
@@ -7,20 +7,36 @@
     <fieldset :disabled="!isEditable && sidebar">
       <div class="row">
           <div :class="{'col-12': sidebar, 'col-6': !sidebar}">
-            <flow-label-editor :flow="flow" />
+
+            <validation-message
+              :message-key="`flow/${flow.uuid}/name` /* `label` is optional, but `name` is required */"
```

Comment:
@bzabos , I'm wondering if you agree with this, or do you have another suggestion?

Reply:
I will leave more explanation at the top

Reply:
Done